.idea/.idea.FSharp.Analyzers.SDK/.idea/.gitignore

```diff
@@ -0,0 +1,13 @@
+﻿# Default ignored files
```

Comment:
Can the `.idea` files/folders be removed from this PR?

Reply:
Done.